components/brave_wallet/renderer/BUILD.gn

```diff
@@ -6,6 +6,8 @@ source_set("renderer") {
     "js_solana_provider.h",
     "resource_helper.cc",
     "resource_helper.h",
+    "safe_builtins.cc",
```

Comment:
seems like this should be a generic helper outside of wallet?

Reply:
If it has to be generic and it needs to include everything. Right now, we only protect the one we used in wallet scripts.

Reply:
well, it needs to include the things we are using, but those can be added for other Brave scripts that use it. What I want to avoid is duplicating this in other places

Reply:
fixed in 9cdc09281c12f97e6e37ac031c4c8572586a2169